lib/index.js

```diff
@@ -1,6 +1,21 @@
 'use strict';
 
-var rollup = require('rollup');
+const fs = require('fs');
+const rollup = require('rollup');
+const debounce = require('debounce');
+
+const dependencies = new Map()
+const changedParents = new Set()
+const WAIT = 25
+
+const touchParents = debounce(() => {
```

Comment:
Don't use const, arrow functions, Map or Set without polyfill or transpilling. It's not so hard to write this stuff in es5.